src/hoverifier.test.ts

```diff
@@ -111,6 +112,94 @@ describe('Hoverifier', () => {
         }
     })
 
+    it('mouseover then click pins the overlay without it disappearing temporarily', () => {
```

Comment:
Please make test descriptions a sentence starting with `it`. "it mouseover then click pins..." doesn't read well